modules/js/index.js

```diff
@@ -0,0 +1,175 @@
+  li.appendChild(removeBookBtn);
+  bookList.appendChild(li);
+
+  removeBookBtn.addEventListener('click', () => {
+    const book = new Book(id, title, author);
+    id = removeBookBtn.id;
+    book.removeBook();
+    if (li.previousElementSibling === null && li.nextElementSibling === null) {
+      bookList.classList.remove('border');
+      li.remove();
+    } else {
+      li.remove();
+    }
+  });
+};
+
+if (books !== null) {
+  books.forEach((book) => {
+    displayBook(book.id, book.title, book.author);
+  });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  addBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    const toTitleCase = (str) => str
+      .toLowerCase()
+      .split(' ')
+      .map((word) => `${word.charAt(0).toUpperCase()}${word.slice(1)}`)
+      .join(' ');
+    const id = Date.now();
+    const book = new Book(id, toTitleCase(title), toTitleCase(author));
+    book.addBook();
+    if (title && author) {
+      displayBook(book.id, book.title, book.author);
+    }
+  });
+});
+
+// navs on clicks
+const bookLists = document.querySelector('.books');
+const addNewBook = document.querySelector('.create');
+const contact = document.querySelector('.contact');
+
+const formContainer = document.querySelector('.form-container');
+const bookContainer = document.querySelector('.main-books');
+const contactContainer = document.querySelector('.contact-container');
+
+bookLists.addEventListener('click', () => {
+  bookContainer.style.display = 'block';
+  formContainer.style.display = 'none';
+  contactContainer.style.display = 'none';
+  bookLists.classList.toggle('blue');
+  addNewBook.classList.remove('blue');
+  contact.classList.remove('blue');
+});
+
+addNewBook.addEventListener('click', () => {
+  formContainer.style.display = 'block';
+  bookContainer.style.display = 'none';
+  contactContainer.style.display = 'none';
+  bookLists.classList.remove('blue');
+  contact.classList.remove('blue');
+  addNewBook.classList.toggle('blue');
+});
+
+contact.addEventListener('click', () => {
+  contactContainer.style.display = 'block';
+  formContainer.style.display = 'none';
+  bookContainer.style.display = 'none';
+  contact.classList.toggle('blue');
+  bookLists.classList.remove('blue');
+  addNewBook.classList.remove('blue');
+});
+
+// date
+// const months = [
+//   'January',
+//   'February',
+//   'March',
+//   'April',
+//   'May',
+//   'June',
+//   'July',
+//   'August',
+//   'September',
+//   'October',
+//   'November',
+//   'December',
+// ];
+// const currentDate = new Date();
+// const date = ` ${
+//   months[currentDate.getMonth()]
+// } ${currentDate.getDate()} ${currentDate.getFullYear()}`;
+// const time = currentDate.toLocaleTimeString();
+// const websiteDate = document.querySelector('.date');
+// websiteDate.innerHTML = `${date} ${time}`;
```

Comment:
- [ ] [OPTIONAL] Because the previous CR didn't mention this it is not a requirement, however, I strongly advise you to remove this piece of commented out code, it will make your code look a lot more professional and readable.